### PR TITLE
CNV-20177: New tekton tasks operator info on architecture page

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.12+

Issue: [CNV-20177](https://issues.redhat.com//browse/CNV-20177)

Link to docs preview: https://52613--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-architecture.html

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: modules/virt-about-tekton-tasks-operator.adoc [hunk 1]
```diff
@@ -9,3 +9,36 @@
...
+|`deployment/cleanup-vm`
+|	Executes a script or a command on a VM, then stops or deletes the VM once the script or command is complete.
+
+|`deployment/disk-virt-customize`
+|	Runs a `customize` script on a target PVC using `virt-customize`.
+
+|`deployment/disk-virt-sysprep`
+|	Runs a `sysprep` script on a target PVC by using `virt-sysprep`.
+
+|`deployment/execute-virt-sysprep`
```
Comment (ksimon1) [inline modules/virt-about-tekton-tasks-operator.adoc:39]: this task does not exist. You can remove it

Comment (ksimon1) [reply modules/virt-about-tekton-tasks-operator.adoc:39]: only execute-virt-sysprep

**Comment by ksimon1** [comment]
The image still contains wrong task

**Comment by sabrinajess** [comment]
Image updated.

**Comment by geetikakay** [comment]
lgtm. sorry for delays.

File: modules/virt-about-tekton-tasks-operator.adoc [hunk 2]
```diff
@@ -9,3 +9,33 @@
...
+|	Copies an {VirtProductName} template.
+
+|`deployment/modify-vm-template`
+|	Creates or removes an {VirtProductName} template.
+
+|`deployment/modify-data-object`
+|	Creates or removes DataVolumes or DataSources.
+
+|`deployment/cleanup-vm`
+|	Executes a script or a command on a VM, then stops or deletes the VM once the script or command is complete.
```
Comment (sjhala-ccs) [inline modules/virt-about-tekton-tasks-operator.adoc:31]: Might want to replace "execute" with "run" 

According to the ISG:
Use with caution
execute   verb
Use only if a simpler term, such as "run", "issue", "start", or "enter" is inappropriate.

Comment (sjhala-ccs) [inline modules/virt-about-tekton-tasks-operator.adoc:31]: Suggestion: s/once the script or command is complete/afterward

File: modules/virt-about-tekton-tasks-operator.adoc [hunk 3]
```diff
@@ -9,3 +9,33 @@
...
+|	Creates a VM from an {VirtProductName} template.
+
+|`deployment/copy-template`
+|	Copies an {VirtProductName} template.
+
+|`deployment/modify-vm-template`
+|	Creates or removes an {VirtProductName} template.
+
+|`deployment/modify-data-object`
+|	Creates or removes DataVolumes or DataSources.
```
Comment (sjhala-ccs) [inline modules/virt-about-tekton-tasks-operator.adoc:28]: s/DataVolumes/data volumes

s/DataSources/data sources

File: modules/virt-about-tekton-tasks-operator.adoc [hunk 4]
```diff
@@ -9,3 +9,33 @@
...
 
 image::cnv_components_tekton-tasks-operator.png[tekton-tasks-operator components]
+
+.tekton-tasks-operator components
+[cols="1,1"]
+|===
+|*Component* |*Description*
+
+|`deployment/create-vm-from-template`
+|	Creates a VM from an {VirtProductName} template.
```
Comment (sjhala-ccs) [inline modules/virt-about-tekton-tasks-operator.adoc:19]: I think "Creates a VM from a template" is more accurate here.

File: modules/virt-about-tekton-tasks-operator.adoc [hunk 5]
```diff
@@ -9,3 +9,33 @@
...
+
+|`deployment/copy-template`
+|	Copies an {VirtProductName} template.
```
Comment (sjhala-ccs) [inline modules/virt-about-tekton-tasks-operator.adoc:22]: "Copies a VM template"

File: modules/virt-about-tekton-tasks-operator.adoc [hunk 6]
```diff
@@ -9,3 +9,33 @@
...
+
+|`deployment/modify-vm-template`
+|	Creates or removes an {VirtProductName} template.
```
Comment (sjhala-ccs) [inline modules/virt-about-tekton-tasks-operator.adoc:25]: "Creates or removes a VM template"

**Review by sjhala-ccs — COMMENTED** [review]
A few nits, otherwise LGTM

**Review by sjhala-ccs — COMMENTED** [review]
LGTM

**Pull request merged**